code-of-conduct-proposed.md

```diff
@@ -52,23 +54,26 @@ Examples of behavior that contributes to a positive environment include:
   and learning from the experience
 * Focusing on what is best not just for us as individuals, but for the
   overall community
+* Using welcoming and inclusive language
+  
 
-Examples of unacceptable behavior include:
+Examples of unacceptable behavior include but are not limited to:
 
 * The use of sexualized language or imagery, and sexual attention or
   advances of any kind
 * Trolling, insulting or derogatory comments, and personal or political attacks
-* Public or private harassment
+* Public or private harassment in any form
+* Comments or jokes based upon stereotypes, that are exclusionary, or that hold others up for ridicule
 * Publishing others' private information, such as a physical or email
   address, without their explicit permission
+* Violent threats
```

Comment:
How about "violence or threats of violence"? Also, note this overlaps with some new language in the other open PR, so although this is a good change it might be modified to avoid adding the same language twice

Reply:
I will change it now! :)